libs/utils/trace.py

```diff
@@ -519,14 +519,19 @@ def _dfg_functions_stats(self, functions=None):
 ###############################################################################
 # Trace Events Sanitize Methods
 ###############################################################################
+    @staticmethod
```

Comment:
I would also make this a public API... it can probably be useful in some analysis methods.